index.bs

```diff
@@ -6668,7 +6674,7 @@ In effect, where <var ignore>x</var> is a <emu-val>Number</emu-val> value,
 
     1.  Let |x| be [=?=] [=ToNumber=](|V|).
     1.  If |x| is <emu-val>NaN</emu-val>, +∞, or −∞,
-        then [=ECMAScript/throw=] a <emu-val>TypeError</emu-val>.
+        then [=ECMAScript/throw=] a {{TypeError}}.
```

Comment:
Well, yes it maps, but what you're effectively throwing here is the ES TypeError, not the WebIDL one. So the conversion is implied, not stated. I think we're loosing clarity by doing this (that is, consistently returning ES values from IDL to ES conversion lowers the cognitive load, imho).